.env.example

```diff
@@ -164,3 +164,5 @@ CSP_POLICY=
 
 # Vercel Edge Config
 EDGE_CONFIG=
+
+NEXT_PUBLIC_MINUTES_TO_BOOK=2 // Minutes
```

Comment:
This could be part of the event config.

Reply:
Also, 5 is a better default, no user is going to finish in 2.